src/features/CourtCaseList/CourtCaseList.tsx

```diff
@@ -20,6 +20,18 @@ const CourtCaseList: React.FC<Props> = ({ csrfToken, courtCases, order = "asc",
   const recentlyUnlockedExceptionId = query.unlockException
   const recentlyUnlockedTriggerId = query.unlockTrigger
 
+  const queryString = Object.entries(query)
+    .reduce((acc, [key, value]) => {
+      if (key === "unlockException" || key === "unlockTrigger") {
```

Comment:
```
      if (key !== "unlockException" && key !== "unlockTrigger") {
            acc.push(`${key}=${value}`)
      }
      return acc
```

What do you think?

Reply:
I debated that myself but preferred this. In my opinion, this is more readable. I find it tricky negating values (more cognitive load).

@richardcane have a preference?

Reply:
@richrace I don't have a strong preference, happy for you to merge it!